.github/actions/install-dashboards/action.yml

```diff
@@ -31,8 +31,8 @@ runs:
     - uses: actions/checkout@v2
       with:
         path: OpenSearch-Dashboards
-        repository: opensearch-project/OpenSearch-Dashboards
-        ref: 'main'
+        repository: wazuh/wazuh-dashboard
+        ref: '2.4.1-init-wzd'
```

Comment:
Does this change have to be merged? Because in the future I think this branch will not exist, and we will have to change it again.